gnmawsgr/tasks.py

```diff
@@ -276,12 +276,7 @@ def do_glacier_restore(request_id,itemid,path):
     while True:
         try:
             with open(filename,'wb') as fp:
-                try:
-                    key.get_file(fp, cb=partial(download_callback, rq), num_cb=40)
-                except AttributeError:
-                    raven_client.captureException()
-                    logger.error("No Key found in S3. Object values: {0}".format(rq.__dict__))
-                    return
```

Comment:
The reason is that this code completely breaks the system. The only thing that will work is a retry, which I assume does not active this broken code.

I assume I got the syntax of this wrong. Can we fix it by removing it for the moment please?